moveit_simple/src/moveit_simple.cpp

```diff
@@ -340,19 +340,20 @@ std::unique_ptr<TrajectoryPoint> Robot::lookupTrajectoryPoint(const std::string
   {
     ROS_INFO_STREAM("Looked up named cart target from robot model: " << name);
     virtual_robot_state_->update();  //Updating state for frame tansform below
-    Eigen::Affine3d pose =  virtual_robot_state_->getFrameTransform(name);
+    Eigen::Affine3d ik_base_to_ik_tip =  virtual_robot_state_->getFrameTransform(name);
+    Eigen::Affine3d pose = srdf_base_to_ik_base_ * ik_base_to_ik_tip;
```

Comment:
I'm not 100% sure about this line where you redefine the pose. It feels like this is a very different definition of what pose used to be, but I'm not familiar with "getFrameTransform" or what uses the pose called by lookupTrajectoryPoint

Reply:
talked with @geoffreychiou and found where srdf_base_to_ik_base_ would get set to identity under the condition where base is aligned with world and then this essentially looks the same as before.